Marble Blast Platinum/platinum/client/scripts/game.cs

```diff
@@ -68,22 +68,60 @@ function clientCmdGameRespawn() {
 }
 
 function updateGameDiscordStatus() {
-	if ($Game::Menu) {
-		// Handled by the UI
-	} else if ($playingDemo) {
-		setDiscordStatus("Watching a Replay");
-	} else {
-		%info = getMissionInfo($Client::MissionFile);
-		%name = %info.name;
+	if (!$Game::Menu) { //UIs handle rich presence on their own
+		%line1 = getMissionInfo($Client::MissionFile).name;
+		if (%line1 $= "Super Secret Puzzle 12") {
+			%line1 = "ZZaZZ ZZdZZiZZmZZhZZ ZZvZZ ZZoZZ ZZ ZZsZZlZZhZZsZZoZZaZZiZZ"; //It's a secret to everybody
```

Comment:
Ok while this is pretty hilarious I think we should pick something sensible here. Maybe along the lines of "A Level", "A Puzzle Level", etc, something that won't draw as much attention lol